service.subtitles.titlovi-com-official/service.py

```diff
@@ -0,0 +1,593 @@
+                        logger('Invalid response status code, exiting!')
+                        return
+                else:
+                    logger('Invalid response status code, exiting!')
+                    show_notification(get_string(32001))
+                    return
+
+                if resp_json['SubtitleResults']:
+                    result_list.extend(resp_json['SubtitleResults'])
+
+            except Exception as e:
+                logger(e)
+                show_notification(get_string(32001))
+                return
+
+            if result_list:
+                addon_cache.set(params_hash, result_list, expiration=timedelta(days=3))
+            else:
+                logger('result_list empty, exiting with error')
+                show_notification(get_string(32001))
+                return
+
+        for result_item in result_list:
+            logger('result_item %s' % result_item)
+            title = result_item['Title']
+            try:
+                season = int(result_item['Season'])
+            except Exception as e:
+                season = -1
+
+            try:
+                episode = int(result_item['Episode'])
+            except Exception as e:
+                episode = -1
+
+            if season > 0 and episode > 0:
+                title = '{0} S{1}E{2}'.format(title, season, episode)
+
+            if result_item['Release']:
+                title = '{0} {1}'.format(title, result_item['Release'])
+
+            listitem = xbmcgui.ListItem(
+                label=result_item['Lang'],
+                label2=title
+            )
+            
+            listitem.setArt({"icon": str(int(result_item['Rating'])), "thumb": LANGUAGE_ICON_MAPPING[result_item['Lang']]})
+            url = "plugin://{0}/?action=download&media_id={1}&type={2}" \
+                .format(script_id, result_item['Id'], result_item['Type'])
+
+            xbmcplugin.addDirectoryItem(handle=plugin_handle, url=url, listitem=listitem, isFolder=False)
+
+    def kodi_load_subtitle(self, subtitle_file_path):
+        logger('loading subtitle: {0}'.format(subtitle_file_path))
+        list_item = xbmcgui.ListItem(label='dummy_data')
+        try:
+            lat_cyr_conversion = int(addon.getSetting('titlovi-lat-cyr-conversion'))
+        except Exception as e:
+            logger(e)
+            lat_cyr_conversion = DONT_CONVERT_LETTERS
+        logger('lat_cyr_conversion: {0}'.format(lat_cyr_conversion))
+        if lat_cyr_conversion != DONT_CONVERT_LETTERS \
+                and '.converted.lat' not in subtitle_file_path\
+                and '.converted.cyr' not in subtitle_file_path:
+            converted_file = handle_lat_cyr_conversion(subtitle_file_path, lat_cyr_conversion)
+            if converted_file is not None:
+                subtitle_file_path = converted_file
+            else:
+                show_notification('Latin/Cyrillic conversion error, showing original subtitle')
```

Comment:
This needs to be localized as it's shown in the ui

Reply:
Fixed